agents-api/agents_api/activities/task_steps/prompt_step.py

```diff
@@ -119,7 +119,7 @@ async def prompt_step(context: StepContext) -> StepOutcome:
     # `function` (see: https://docs.litellm.ai/docs/providers/anthropic#computer-tools)
     # but we don't allow that (spec should match type).
     formatted_tools = []
-    for i, tool in enumerate(context.tools):
+    for i, tool in enumerate(await context.tools):
```

Comment:
should refactor `.tools` getter to an async method. this is weird

Reply:
yeah...